control-plane/README.md

```
@@ -0,0 +1,121 @@
+2.  `helm` installed.
+3.  `kubectl` configured.
+
+## 1. Deploy Observability Stack (Loki + Grafana)
+
+We use the official `grafana/loki-stack` chart.
+
+```bash
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+
+helm upgrade --install observability grafana/loki-stack \
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)